Makefile

```diff
@@ -1,8 +1,7 @@
-.SHELLFLAGS := -eu -o pipefail -c
 MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 
-TAG ?= $(shell git fetch --tags && git describe --tags --abbrev=0 | sed 's/v//')
```

Comment:
Missing `shell` after `$(`

Reply:
Doh!  Replaced the shell.

I do have my config set up to always do ssh.  Anyway, as you point out, make targets are used purely as utility by dev...  so I'll just put that back as well.